website/src/pages/showcase.module.css

```diff
@@ -0,0 +1,338 @@
+/**
```

Comment:
Lots of copy&paste. Please remove unrelevant parts.

Reply:
I cleaned it up and moved the @media queries to the bottom where they normally are seen.